cmd/flags.go

```diff
@@ -40,6 +40,8 @@ const (
 	flagVersion                        = "version"
 	flagEnableDebugServer              = "enable-debug-server"
 	flagDebugAddr                      = "debug-addr"
+	flagEnableMetricsServer            = "enable-metrics-server"
+	flagMetricsAddr                    = "metrics-addr"
```

Comment:
UX: if a user sets `metrics-addr`, wouldn't this show their intent to enable it as well?

Either
- if `flagMetricsAddr` is set but `flagEnableMetricsServer` is false, notify the user
- if `flagMetricsAddr` is set, set `flagEnableMetricsServer` to true.

Reply:
If you set `--metrics-addr` but do not set `--enable-metrics-server` - should the relayer display a warning and start or should it refuse to start with a warning?

Reply:
display a warning and start

Reply:
imo we should remove the flags for explicitly setting the debug and metrics addresses. we should keep the `enable` flags and then just consume the addresses from the config file.

Reply:
Is it ever common to run multiple rly instances on the same host? If so, do people use different home directories or share the same?

Reply:
that's a good question, in theory i wouldn't imagine so since one rly instance can handle many paths. if they were running multiple instances i would imagine each would have their own config file so it should be okay to configure the server addresses via the config file. if there were multiple rly instances using one config file then they would end up using the same keys which would result in a ton of account sequence errors.

Reply:
removed flags for addresses and added warnings if you enable server but the address setting is missing in the config.

Reply:
Interchain test depends on --debug-addr https://github.com/cosmos/relayer/blob/4e4e9530800d28fb2c984f1cfc7b03f05eec618c/interchaintest/relayer.go#L293

Reply:
so I think we should keep the flags for the addresses.

Reply:
I improved logs so when you provide address flags but you do not enable the server you will get a warning.